elasticsearch/values.yaml

```diff
@@ -68,7 +68,7 @@ podAnnotations: {}
 # additionals labels
 labels: {}
 
-esJavaOpts: "-Xmx1g -Xms1g"
+esJavaOpts: "" # example: "-Xmx1g -Xms1g"
```

Comment:
I'm not sure this correctly formatted, should the whole `esJavaOpts` line be commented out to have it not be set by default?